### ISSUE TITLE
Files, that do not differ in size, are not synced

### ISSUE THREAD

**Issue by rtrn** [ncw/rclone#911]
I'm using rclone 1.34 on Microsoft Windows.  My storage provider is Yandex with Crypt on top.

Syncing works.  Except that changed files, which do not differ in size, are not picked up.  Neither by `rclone check`, nor by `rclone sync`.  It seems that modification times and checksums are ignored, and only sizes are compared.

**Comment by docloulou** [comment]
try these:
```
--ignore-size                       Ignore size when skipping use mod-time or checksum.
  -I, --ignore-times                      Don't skip files that match size and time - transfer all files
```

**Comment by rtrn** [comment]
I tried `--ignore-size` and `--checksum` and both don't change anything.
`--ignore-times` syncs all files every time, which is unacceptable for me.

**Comment by ncw** [comment]
checksum isn't supported over crypt - see #637 for the ultimate solution.

Also see #854

Modtime should be supported with Yandex though, can you give log of it not working please with `-v` please?

**Comment by rtrn** [comment]
This definitely looks like a bug.  Rclone picks up the changed modification time, but decides to skip the file nonetheless.
Here is the transcript:
```
$ echo foo >foo/foo.txt
$ ./rclone.exe --config rclone.conf -v sync foo secret:
2016/11/27 21:07:21 rclone: Version "v1.34" starting with parameters ["C:\\Users\\dfe\\Desktop\\rclone\\rclone.exe" "--config" "rclone.conf" "-v" "sync" "foo" "secret:"]
2016/11/27 21:07:22 Encrypted Yandex secret: Modify window is 100ns
2016/11/27 21:07:22 Encrypted Yandex secret: Waiting for checks to finish
2016/11/27 21:07:22 Encrypted Yandex secret: Waiting for transfers to finish
2016/11/27 21:07:24 foo.txt: Copied (new)
2016/11/27 21:07:24 Waiting for deletions to finish
2016/11/27 21:07:24
Transferred:      4 Bytes (1 Bytes/s)
Errors:                 0
Checks:                 0
Transferred:            1
Elapsed time:        3.2s
2016/11/27 21:07:24 Go routines at exit 3

$ echo bar >foo/foo.txt
$ ./rclone.exe --config rclone.conf -v sync foo secret:
2016/11/27 21:08:25 rclone: Version "v1.34" starting with parameters ["C:\\Users\\dfe\\Desktop\\rclone\\rclone.exe" "--config" "rclone.conf" "-v" "sync" "foo" "secret:"]
2016/11/27 21:08:26 Encrypted Yandex secret: Modify window is 100ns
2016/11/27 21:08:26 Encrypted Yandex secret: Waiting for checks to finish
2016/11/27 21:08:26 foo.txt: Modification times differ by -1m35.6710648s: 2016-11-27 21:08:11.4751257 +0100 CET, 2016-11-27 21:06:35.8040609 +0100 CET
2016/11/27 21:08:26 foo.txt: Size of src and dst objects identical
2016/11/27 21:08:26 foo.txt: Unchanged skipping
2016/11/27 21:08:26 Encrypted Yandex secret: Waiting for transfers to finish
2016/11/27 21:08:26 Waiting for deletions to finish
2016/11/27 21:08:26
Transferred:      0 Bytes (0 Bytes/s)
Errors:                 0
Checks:                 1
Transferred:            0
Elapsed time:        1.6s
2016/11/27 21:08:26 Go routines at exit 1
```

The `rclone.conf` looks like this:
```
[remote]
type = yandex
client_id =
client_secret =
token = {"access_token":"XXX","token_type":"bearer","refresh_token":"XXX","expiry":"2017-11-27T21:02:34.4970515+01:00"}

[secret]
type = crypt
remote = remote:secret
filename_encryption = standard
password = XXX
password2 = XXX

```

**Comment by ncw** [comment]
i confirm that this is a bug.

I've fixed it in 2756900749a37d450a1c31fab6386c027333844e

Here is a beta with the fix: http://beta.rclone.org/v1.34-37-g2756900/ (uploaded in 15-30 mins)

Please re-open if there is any problems.
